source/standards/understanding-risks.html.md.erb

```diff
@@ -0,0 +1,43 @@
+---
+title: Understand the risks to your service
+expires: 2019-03-01
+---
+
+# <%= current_page.data.title %>
+
+<%= partial :expires %>
+
+When you build, maintain or change your service you must have a clear understanding of any associated threats and risks because they will impact your service design.
+
+You should work with the [GDS Information Assurance (IA)][] team to understand your service’s risks and design appropriate solutions. In some scenarios IA may need to obtain risk acceptance from your Senior Information Risk Owner (SIRO) before you can go ahead with certain activities.
```

Comment:
When we mention 'in some scenarios' and 'with certain activities' can we give an example or two?